src/cli.js

```diff
@@ -33,6 +33,7 @@ const main = () => {
     .arguments("[<json>]")
     .option("-o, --output <format>", `Format of the output: ${available}`, "md")
     .option("-t, --template <template>", "Template to use for rendering")
+    .option("-f, --file <file>", "Read an existing JSON file")
```

Comment:
I don't think we need `<file>` here, just a flag to inform the app about what format to expect as input. So if the flag is simply `--file`, we know we can interpret `<json>` (from line 33) as a file path.

Moreover, I am working on something similar (a flag to identify the sort of input) in this WIP branch (probably PR coming later today). I suggested, there, `--ckan` to tell the app the input is a CKAN Schema, not a JSON Schema (as described in #13). Maybe we can replace `--file` and `--ckan` by a more general `--input <type>` having `--input json` as a default, and leaving room for us to implement `--input path` and `--input ckan`. What do you think?

Reply:
Sounds better using the `--input` flag. Do you need any help? We can chat on Telegram if you need!